docs/guide/java_object_graph_guide.md

```diff
@@ -121,8 +121,8 @@ Thread-safe fury:
 ```java
 ThreadSafeFury fury=Fury.builder()
   .withLanguage(Language.JAVA)
-  // enable referecne tracking for shared/circular reference.
```

Comment:
Thanks for taking time to contribute to this project. There are similar typos  in #L105 for `referecne` and #L106 for `duplciate` , would you mind fixing it too?